definitions/ext-firewall/summary_metrics.yml

```diff
@@ -83,7 +71,7 @@ sessionsTotal:
       eventId: entity.guid
     # Fortinet Fortigate firewalls
     kentik/fortinet-fortigate:
-      select: max(kentik.snmp.fgSysSesCount + kentik.snmp.fgSysSes6Count)
```

Comment:
could this be rewritten as `max(a + b)` instead of `max(a) + max(b)`? I believe the current query might lead to some issues on our side